.github/workflows/points-next.yml

```diff
@@ -0,0 +1,55 @@
+name: Points with next Rails
+on:
+  push:
+    branches:
+    - main
+  pull_request:
+    branches:
+    - main
+
+jobs:
+  build-rails-next:
+    runs-on: ubuntu-latest
+    
+    env:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
+      RAILS_ENV: test
+    
+    services:
+      postgres:
+        image: postgres:11.6-alpine
+        ports:
+          - 5432:5432
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+        # needed because the postgres container does not provide a healthcheck
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v1
+
+      - name: Install PostgreSQL 11 client
+        run: |
+          sudo apt-get -yqq install libpq-dev
+
+      - name: Setup Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: "3.0.2"
+
+      - name: Build App
+        run: |
+          bundle install
+          yarn install
```

Comment:
we can use a github action that will cache the result of yarn install, check the one I used here https://github.com/arielj/vanilla-nested/blob/main/.github/workflows/main.yml

Reply:
@arielj : That step requires `package-lock.json` to be available in the repository. We don't have `yarn.lock` either. I wander, if we really need `yarn install` here.

Reply:
@neutrino I think we can ignore this in this repo, I'm checking the original CircleCI config and it's not running `yarn install` and the package.json is just empty so running `yarn install` is useless, let's remove that